### PR TITLE
finish kanye records challenge for week two

### PR DESCRIPTION
**Pull request by BrianZCheung**
This PR completes the kanye records code challenge for week two of the code challenges.

### CODE REVIEW

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 1]
```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |hash_iterate| puts hash_iterate[:title] end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: Nothing is technically stopping you from putting this code on one line, but generally if you're using `do/end` then the code should be on three lines.

You could reformat to use the `item.each { |x| <block> }` syntax - that's generally acceptable to keep on one line.

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 2]
```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
...
-# write your code here
+kanyeRecords.each do |hash_iterate| puts hash_iterate[:year] end
 ```
 
 3. Look into the future and add a forthcoming Kanye record into our array. It should be released in 2021 and have a title that is completely asinine.
 
 ```ruby
-# write your code here
+kanyeRecords << {year: 2021, title: "Tis the Season to dress it salty"}
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md:35]: Nice! 🌶 🎄

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 3]
```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
...
 
 6. Iterate through collection. add a key value pair to each record with the key as `label` and the value as `"def jam"`.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |i|
+  i['label'] = "def jam"
+end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: Remember, be consistent within your data structure. You're using symbols for `year` and `title`, so you should use a symbol for the `label` as well.

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 4]
```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
...
 ```ruby
-# write your code here
+kanyeRecords.each do |i|
+  if i[:title].length >= 15
+    puts "'#{:title}' That's a long title"
+  else
+    puts "'#{:title}' That's a short title"
+  end
+end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: I think you need to revisit your interpolation here. This is the current output:
```
'title' That's a long title
'title' That's a short title
'title' That's a long title
'title' That's a long title
'title' That's a long title
'title' That's a short title
'title' That's a long title
'title' That's a long title
'title' That's a long title
```

File: brian_cheung/week_two_challenges/kanye_records.md (hunk 1 above)
Comment (BrianZCheung) [reply brian_cheung/week_two_challenges/kanye_records.md]: oh, thats how you do it with one line..

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 5]
```diff
@@ -19,13 +19,13 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-kanyeRecords.each do |hash_iterate| puts hash_iterate[:title] end
+kanyeRecords.each { |hash_iterate| puts hash_iterate[:title] }
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: This `hash_iterate` value doesn't have the best name. What is "each" thing here that has a title?

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 6]
```diff
@@ -67,9 +63,9 @@ end
...
 kanyeRecords.each do |i|
   if i[:title].length >= 15
-    puts "'#{:title}' That's a long title"
+    puts "'#{i[:title]}' That's a long title"
   else
-    puts "'#{:title}' That's a short title"
+    puts "'#{i[:title]}' That's a short title"
   end
 end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: Again with the naming of `i` here - you should aim to give your variables names that clarify what they actually represent.

File: brian_cheung/week_two_challenges/kanye_records.md [hunk 7]
```diff
@@ -19,13 +19,13 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-kanyeRecords.each { |hash_iterate| puts hash_iterate[:title] }
+kanyeRecords.each { |each_kanyeRecords_hash| puts each_kanyeRecords_hash[:title] }
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/kanye_records.md]: Now this seems excessive - how about simply, `record`?

**Review by gretchenziegler — APPROVED** [review]
🕺

**Pull request merged**